pulpcore/pulpcore/app/models/publication.py

```diff
@@ -1,17 +1,39 @@
-from django.db import models
+from django.db import models, transaction
 
-from pulpcore.app.models import Model, storage
+from . import storage
+from .base import Model
+from .task import CreatedResource
 
 
 class Publication(Model):
     """
+    A publication.
```

Comment:
From the language guide: 
Publications refer to the metadata and arifacts of the content units in a repository version.